src/Microsoft.Diagnostics.Tools.RuntimeClient/Eventing/EventPipeClient.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.IO.Pipes;
+using System.Linq;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+using System.Security.Principal;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Diagnostics.Tools.RuntimeClient.Eventing
```

Comment:
I'd suggest eliminating the "Eventing" at the end. This is fairly subjective, but there were a few reasons why I lean that way:
a) The namespace name is already quite long
b) Although namespaces can be made arbitrarily precise, I try not to go further than necessary to eliminate naming collisions. I'm predicting nothing in EventPipe will collide with other profiler or debugger names.
c) Its easier for users to guess the right namespace name if it matches the assembly name, and I am suggesting that we only have one assembly for all the RuntimeClient APIs.